src/BuildIt.Forms/BuildIt.Forms.Controls/Common/Strings.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace BuildIt.Forms.Controls.Common
+{
+    /// <summary>
+    /// Class containing all of the strings
+    /// </summary>
+    internal class Strings
```

Comment:
Do we not already have a Constants class in Forms.Controls? Even if we don't, can we follow what we use elsewhere which is a class called Constants, and then inner classes for different categories of constants

Reply:
There's no Constants class. 

I'm fine with renaming this class to be Constants. I just wonder though, as I tend to create Constants and Strings classes in different projects. Is there a particular reason why you want to keep the string literals in Constants not in Strings?

Reply:
More of a personal preference than anything but happens to align with the rest of the BuildIt libraries. For most projects I don't really care, so long as it's consistent across that project

Reply:
Sure, will change it